test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -639,7 +640,8 @@ private void updateTime(final long leftTime) {
             long hours = leftTime / 3_600_000;
             long minutes = (leftTime - hours * 3_600_000) / 60_000;
             long seconds = (leftTime - hours * 3_600_000 - minutes * 60_000) / 1_000;
-            label.setText(String.format("Test timeout: %02d:%02d:%02d",
+            label.setText(String.format(Locale.ENGLISH,
+                                        "Test timeout: %02d:%02d:%02d",
```

Comment:
is this the only place where we must enforce the locale?  Surely, there must be many, many other occurrences!

Reply:
In the `PassFailJFrame` class, there's no more code which depends on locale. This is the only place where something is formatted.

Other tests, including those that use the `PassFailJFrame` framework, may depend on the current locale. It is not a goal to update all possible tests which may fail in non-English locale to use an English locale explicitly.

There are tests which explicitly set the locale, for example [`PageDialogMarginTest.java`](https://github.com/openjdk/jdk/blob/b938a5c9edd53821a52b43a8e342b76adb341a3f/test/jdk/java/awt/print/PrinterJob/PageDialogMarginTest.java#L45).